src/pages/studyView/StudyViewPage.tsx

```diff
@@ -329,8 +330,31 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                                                     currentTab={this.store.currentTab}
                                                     addChartOverlayClassName='studyViewAddChartOverlay'
                                                     disableCustomTab={this.store.currentTab === StudyViewPageTabKeyEnum.CLINICAL_DATA}
+                                                    showResetPopup={() => { this.showModal = true; }}
                                                 />
                                         )}
+
+                                        <Modal bsSize={'small'} show={this.showModal} onHide={() => { this.showModal = false; }} keyboard>
+                                            <Modal.Header closeButton>
+                                                <Modal.Title>Reset to Default view</Modal.Title>
+                                            </Modal.Header>
+                                            <Modal.Body>
+                                                <div>Are you sure you want to override the list and order of the selected charts
+                                                     with the default setting? This is irreversible.</div>
+                                            </Modal.Body>
+                                            <Modal.Footer>
+                                                <button className='btn btn-primary btn-sm'
```

Comment:
User could exit the modal when decided not to restore by clicking the x icon on the top right. I think it also make sense to have a `No` button after `Yes` to naturally close the dialog.